sdk/core/ts-http-runtime/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 0.3.0 (Unreleased)
```

Comment:
Since the change is additive I think we should keep this as 0.2.2 (reserving minor version bumps for breaking changes pre-1.0). Since major version 0 is special `^0.2.0` will only pull in anything below `0.3.0`, so bumping the minor version here will mean that this won't get pulled in automatically

Reply:
Sounds good!